sermant-plugins/sermant-springboot-registry/springboot-registry-plugin/src/main/java/io/sermant/discovery/config/NacosRegisterConfig.java

```diff
@@ -0,0 +1,402 @@
+/*
+ * Copyright (C) 2024-2024 Sermant Authors. All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package io.sermant.discovery.config;
+
+import io.sermant.core.common.LoggerFactory;
+import io.sermant.core.config.ConfigManager;
+import io.sermant.core.config.common.ConfigTypeKey;
+import io.sermant.core.plugin.config.PluginConfig;
+import io.sermant.core.plugin.config.PluginConfigManager;
+import io.sermant.core.plugin.config.ServiceMeta;
+import io.sermant.core.utils.AesUtil;
+import io.sermant.core.utils.StringUtils;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.Properties;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * nacos Register plug-in configuration
+ *
+ * @author xiaozhao
+ * @since 2024-11-16
+ */
+@ConfigTypeKey(value = "nacos.service")
```

Comment:
The value of @ConfigTypeKey can not be the same as other plugins.  I read the code and find that currently When use ConfigManager.getConfig(NacosRegisterConfig .class); It may get wrong config